src/lib/interfaces.ts

```diff
@@ -52,43 +52,44 @@ export interface ActivateJobsRequest {
 
 export interface ActivatedJob {
```

Comment:
If you want to make all fields as readonly, I recommends you to remove all readonly key and use Readonly<ActivatedJob> instead.